graf2d/gpad/inc/TRatioPlot.h

```diff
@@ -135,59 +136,60 @@ class TRatioPlot : public TPad {
+   Bool_t IsDrawn();
 
-   virtual void Init(TH1* h1, TH1* h2, Option_t *option = "", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP");
+   virtual void Init(TH1* h1, TH1* h2, Option_t *option = "");
 
 public:
 
    TRatioPlot();
    virtual ~TRatioPlot();
-   TRatioPlot(TH1* h1, TH1* h2, Option_t *option = "pois", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP", const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
-
-   TRatioPlot(THStack* st, TH1* h2, Option_t *option = "pois", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP", const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
+   TRatioPlot(TH1* h1, TH1* h2, Option_t *option = "pois");
 
-   TRatioPlot(TH1* h1, Option_t *option = "", Option_t *h1DrawOpt = "", Option_t *graphDrawOpt = "LX", TFitResult *fitres = 0, const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
+   TRatioPlot(THStack* st, TH1* h2, Option_t *option = "pois");
 
+   TRatioPlot(TH1* h1, Option_t *option = "", TFitResult *fitres = 0);
 
+   void SetH1DrawOpt(Option_t *opt);
+   void SetH2DrawOpt(Option_t *opt);
+   void SetGraphDrawOpt(Option_t *opt);
 
    virtual void Draw(Option_t *chopt="");
    virtual void Browse(TBrowser *b);
 
    virtual void BuildLowerPlot();
 
    virtual void Paint(Option_t *opt = "");
-   virtual void PaintModified();
 
    // Slots for signal receiving
-   virtual void UnZoomed();
-   virtual void RangeAxisChanged();
-   virtual void SubPadResized();
+   void UnZoomed();
+   void RangeAxisChanged();
+   void SubPadResized();
 
    // Getters
-   virtual TAxis *GetXaxis() { return fSharedXAxis; }
-   virtual TAxis *GetUpYaxis() { return fUpYaxis; }
-   virtual TAxis *GetLowYaxis() { return fLowYaxis; }
+   TAxis *GetXaxis() { return fSharedXAxis; }
+   TAxis *GetUpYaxis() { return fUpYaxis; }
+   TAxis *GetLowYaxis() { return fLowYaxis; }
 
    virtual TGraph *GetLowerRefGraph();
-   virtual TAxis *GetLowerRefXaxis();
-   virtual TAxis *GetLowerRefYaxis();
+   TAxis *GetLowerRefXaxis();
+   TAxis *GetLowerRefYaxis();
 
    virtual TObject *GetUpperRefObject();
-   virtual TAxis *GetUpperRefXaxis();
-   virtual TAxis *GetUpperRefYaxis();
+   TAxis *GetUpperRefXaxis();
+   TAxis *GetUpperRefYaxis();
 
-   virtual TPad * GetUpperPad() { return fUpperPad; }
-   virtual TPad * GetLowerPad() { return fLowerPad; }
+   TPad * GetUpperPad() { return fUpperPad; }
+   TPad * GetLowerPad() { return fLowerPad; }
 
    // Setters
-   virtual void SetFitResult(TFitResultPtr fitres);
-   virtual void SetFitResult(TFitResult *fitres); 
+   void SetFitResult(TFitResultPtr fitres);
+   void SetFitResult(TFitResult *fitres); 
```

Comment:
I found it unintuitive to use TFitResultPtr, and I cannot just use TFitResult* since TFitResultPtr does not convert to TFitResult* automatically.